changelog/unreleased/issue-3414

```diff
@@ -1,19 +1,19 @@
-Enhancement: Add --keep-within-hourly switch to restic forget
+Enhancement: Add `--keep-within-hourly` option to restic forget
 
-restic forget allowed users to specify keeping a given number of 
+Restic forget allowed users to specify keeping a given number of
 hourly backups, or to keep all backups within a given interval,
 but not both, that is to specify keeping hourly backups within
 a given interval.
```

Comment:
Suggest "The `forget` command allowed keeping a given number of hourly backups or to keep all backups within a given interval, but it was not possible to specify keeping hourly backups within a given interval.".